macos/Sources/Features/QuickTerminal/QuickTerminalController.swift

```diff
@@ -390,25 +393,32 @@ class QuickTerminalController: BaseTerminalController {
 
         // Update our derived config
         self.derivedConfig = DerivedConfig(config)
-
+        self.position = self.derivedConfig.quickTerminalPosition
+        
         syncAppearance(config)
     }
 
     private struct DerivedConfig {
         let quickTerminalScreen: QuickTerminalScreen
         let quickTerminalAnimationDuration: Double
         let quickTerminalAutoHide: Bool
+        let quickTerminalPosition: QuickTerminalPosition
+        let quickTerminalSize: QuickTerminalSize
```

Comment:
```suggestion
        let quickTerminalPosition: QuickTerminalPosition
        let quickTerminalSize: QuickTerminalSize
        let windowColorspace: String
        let backgroundOpacity: Double
```

or your preferred order